force-app/main/default/classes/EncryptionRecipesTest.cls

```diff
@@ -0,0 +1,229 @@
+@isTest
+private with sharing class EncryptionRecipesTest {
+    @isTest
+    static void encryptAndDecryptWithManagedIVRecipesTest() {
+        // GIVEN
+        String toEncrypt = 'Test data';
+        Blob dataToEncrypt = Blob.valueOf(toEncrypt);
+
+        // WHEN
+        Test.startTest();
+        Blob encryptedData = EncryptionRecipes.encryptWithManagedIVRecipe(
+            dataToEncrypt
+        );
+        Blob decryptedData = EncryptionRecipes.decryptWithManagedIVRecipe(
+            encryptedData
+        );
+        Test.stopTest();
+
+        // THEN
+        System.assertEquals(
```

Comment:
Similar comment to above regarding constant time comparison. The impact for tests is much lower, but the code should be updated to be consistent with above comparison advice